src/ClockFactory.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+
+namespace Cake\Chronos;
+
+/**
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @copyright     Copyright (c) Daniel Opitz
```

Comment:
We don't need  separate copyright, just the cakephp one.

Reply:
I added this, because it was also added here:

https://github.com/cakephp/chronos/blob/3.x/src/Chronos.php#L11
https://github.com/cakephp/chronos/blob/3.x/src/DifferenceFormatter.php#L11

Maybe we remove the other redundant copyright tags as well?

Reply:
The very original Chronos was forked from a different project so Mark inherited a copyright for some of the files. It's very close to not applying now, but definitely not new code.